raccoon/layers/reccurrent.py

```
@@ -60,7 +60,7 @@ def __init__(self, ls_n_in, n_out, initializer, grad_clipping=None):
     def precompute_inputs(self, inputs):
         return T.dot(inputs, self.w_in) + self.b_in
 
-    def step(self, inputs, h_pre, mask=None, process_inputs=False):
+    def step(self, inputs, h_pre, mask=None, process_inputs=True):
```

If process_inputs=False, then default behavior of step is different from default behavior of apply, which may lead to misleading results, if one is not very careful.